src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+
+import { User } from '../../models/user';
+import { AppState } from '../../../store/states/app.states';
+import { LogIn } from '../../../store/actions/account.actions';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.css'
+})
+export class LoginComponent implements OnInit {
+
+  user: User = new User();
+
+  constructor(
+    private store: Store<AppState>
+  ) {}
+
+  ngOnInit() {
+  }
+
+  onSubmit(): void {
+    const payload = {
+      email: this.user.email,
+      password: this.user.password
+    };
+    this.store.dispatch(new LogIn(payload));
```

Comment:
try to use new format of ngRx https://ngrx.io/guide/store/actions

Reply:
Fixed